internal/db/rdb/mysql/mysql.go

```diff
@@ -305,25 +304,25 @@ func (m *mySQLClient) SetMetas(ctx context.Context, metas ...MetaVector) error {
 	defer tx.RollbackUnlessCommitted()
 
 	for _, meta := range metas {
-		err = validateMeta(meta)
+		err = validateVector(meta)
 		if err != nil {
 			return err
 		}
 
-		_, err = tx.InsertBySql("INSERT INTO meta_vector(uuid, vector, meta) VALUES (?, ?, ?) ON DUPLICATE KEY UPDATE vector = ?, meta = ?",
+		_, err = tx.InsertBySql("INSERT INTO "+vectorTableName+"(uuid, vector, meta) VALUES (?, ?, ?) ON DUPLICATE KEY UPDATE vector = ?, meta = ?",
```

Comment:
same
```suggestion
		_, err = tx.InsertBySql("INSERT INTO "+vectorTableName+"(uuid, vector) VALUES (?, ?) ON DUPLICATE KEY UPDATE vector = ?",
```
also remove L315 and L316